docker/api/image.py

```diff
@@ -273,6 +273,7 @@ def load_image(self, data):
         """
         res = self._post(self._url("/images/load"), data=data)
         self._raise_for_status(res)
+        return self._result(res, True)
```

Comment:
Ah but in #1667 it doesn't return the result, it only raises (which is the point of my PR). Either you merge #1667 first and I fix the conflict or you can update #1667 by replacing the `self._raise_for_status` by the `self._result`.